### PR TITLE
create product collection category and move to main docs folder

### PR DESCRIPTION
**Pull request by piinthecloud**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1.
2.
3.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: docs/product-collection-block/register-product-collection.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
+---
+post_title: Registering the product collection block
+menu_title: Registering the Product Collection Block
```
Comment (kmanijak) [inline docs/product-collection-block/register-product-collection.md]:
```suggestion
post_title: Registering custom collections in product collection block
menu_title: Registering custom collections in Product Collection Block
```

File: docs/product-collection-block/register-product-collection.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
+---
+post_title:  Registering custom collections in product collection block
```
Comment (kmanijak) [inline docs/product-collection-block/register-product-collection.md]: Nitpick, double space 🙈 

```suggestion
post_title: Registering custom collections in product collection block
```

**Review by kmanijak — APPROVED** [review]
Thanks @piinthecloud for this example! Left one minor nitpick but preapproving anyway! 🚀

**Review by shaybanshee — COMMENTED** [review]
Works on develop!

**Pull request merged**